providers/implementations/kdfs/hkdf.c

```diff
@@ -149,6 +149,7 @@ static int kdf_hkdf_derive(void *vctx, unsigned char *key, size_t keylen,
 
     switch (ctx->mode) {
     case EVP_KDF_HKDF_MODE_EXTRACT_AND_EXPAND:
```

Comment:
This line becomes unnecessary with the move of _default:_

I'm not entirely sure it's a good idea -- checking arguments is considered good.  Not enough to insist on a change though.

Reply:
The default will never get hit.. But if I take it out something is likely to complain.

Reply:
I was thinking, error if unknown like it did before.